ipsec-vpn/hier/usr/lib/python3/dist-packages/tests/test_ipsec_vpn.py

```diff
@@ -840,12 +844,14 @@ def test_082_any_remote_tunnel_ping(self):
         events = global_functions.get_events("IPsec VPN",'Tunnel Connection Events',None,5)
         found = global_functions.check_events( events.get('list'), 5, 
                                               "event_type", "UNREACHABLE" )
+        # set to original settings
+        self._app.setSettings(org_ipsec_settings)
```

Comment:
The remote NGFW needs to be restored at this point
 remote_app.setSettings(org_remote_ipsec_settings)

Reply:
resolved